core-build/gulp-core-build/package.json

```diff
@@ -33,11 +33,11 @@
     "gulp": "~4.0.2",
     "gulp-flatten": "~0.2.0",
     "gulp-if": "^2.0.1",
-    "@jest/core": "25.4.0",
-    "@jest/reporters": "25.4.0",
-    "jest": "25.4.0",
-    "jest-cli": "25.4.0",
-    "jest-environment-jsdom": "25.4.0",
+    "@jest/core": "~25.4.0",
+    "@jest/reporters": "~25.4.0",
+    "jest": "~25.4.0",
+    "jest-cli": "~25.4.0",
+    "jest-environment-jsdom": "~25.4.0",
```

Comment:
Be careful -- Jest's [massive dependency tree](https://npm.anvaka.com/#!/view/2d/jest) is pretty brittle already with `rush update --full`

Reply:
It may make sense to put the Jest plugin in its own package. Thoughts on that?

Reply:
Yes - we have a lot of projects that use Jasmine for example, which does seem to have an upside of running faster than Jest. But let's get Heft's basic task set working first, then we can make some judicious decisions about where the package boundaries should be.